### ISSUE TITLE
[Dialog] Add responsiveness to Modal Dialogs

### ISSUE THREAD

**Issue by ramtinabadi** [Vieolo/vieolo-ui#128]
The modal dialogs should have the ability to turn into a full screen page in tablet and/or mobile layout, depending on the received props
